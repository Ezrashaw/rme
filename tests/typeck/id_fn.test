fn id(x) = x;
id(5);
id(true);