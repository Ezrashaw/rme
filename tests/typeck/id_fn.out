[36;1mfn[0m id[95m([0mx[95m)[0m[90;1m: [0mfn(?a) -> ?a[90;1m = [0mx
id[95m([0m[33;3m5[0m[95m)[0m[90;1m: [0mfloat
